Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:10-alpine
+FROM node:alpine3.11
```

Comment:
@DorTruzman what is the recommended version of node? What is your version?

Reply:
The newest version should be fine